tests/recovery.py

```diff
@@ -489,9 +441,7 @@ def run(args):
                     network, args, from_snapshot=False
                 )
             else:
-                network = test_recover_service(
-                    network, args, from_snapshot=False, split_ledger=True
-                )
+                network = test_recover_service(network, args, from_snapshot=False)
 
             for node in network.get_joined_nodes():
                 node.verify_certificate_validity_period()
```

Comment:
Can you also remove the comment on line `496` and the test-specific arguments that were only set because of this?

Reply:
Of course, thanks for pointing that out!